src/js/messaging/containers/Compose.jsx

```diff
@@ -24,13 +26,19 @@ import {
   fetchSenderName
 } from '../actions/compose';
 
+import {
+  toggleConfirmDelete
+} from '../actions/modals';
+
 class Compose extends React.Component {
   constructor() {
     super();
     this.handleCategoryChange = this.handleCategoryChange.bind(this);
+    this.handleMessageChange = this.handleMessageChange.bind(this);
     this.handleRecipientChange = this.handleRecipientChange.bind(this);
     this.handleSubjectChange = this.handleSubjectChange.bind(this);
-    this.handleMessageChange = this.handleMessageChange.bind(this);
+    this.handleDeleteDraft = this.handleDeleteDraft.bind(this);
+    this.handleDelete = this.handleDelete.bind(this);
```

Comment:
`handleDeleteDraft` triggers the confirmation modal. `handleDelete` handles the actual deletion. Removed `handleDeleteDraft` in favor of using `toggleDeleteConfirm` in both places.